src/templates/CourseInfoTemplate.js

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from "react"
+import { graphql } from "gatsby"
+import styled from "styled-components"
+import rehypeReact from "rehype-react"
+import { Helmet } from "react-helmet"
+
+import Layout from "./Layout"
+
+import getNamedPartials from "../partials"
+
+import "./remark.css"
+import { LoginStateContextProvider } from "../contexes/LoginStateContext"
+import Container from "../components/Container"
+import Banner from "../components/Banner"
+import PagesContext from "../contexes/PagesContext"
+
+const ContentWrapper = styled.article``
+
+export default class CourseInfoTemplate extends React.Component {
+  render() {
+    const { data } = this.props
+    const { frontmatter, htmlAst } = data.page
+    const allPages = data.allPages.edges.map((o) => {
```

Comment:
Should filter out pages that have "hidden: true" in their metadata.

Because the siderbar navigation in the course material only navigates to the index page for each part, teachers usually only apply this hidden boolean in the `index.md` file. So somehow we need to check if the `/osa-01/index.md` has the hidden flag set to true, we also should hide all sub pages and exercises for that part in this new template.

Reply:
Implemented in b48ed9f. Now lists by overview page with proper handling of hidden overview pages.